### DISCUSSION TITLE
Red4ext does not work on Linux (POP OS 22.04 LTS)

### DISCUSSION THREAD

**Discussion by jceloto7** [wopss/RED4ext#19] [Q&A] [answered]
Hey guys

I'm a Linux user and I'm having problems with RED4ext.

Even after following the installation steps, the program doesn't even start. I was able to observe this because there is not even a log file, as it should have according to the documentation.

I had a very similar problem with Cyber ​​Engine Tweaks and I was able to solve it using Proton Tricks.

How can I make this mod work on my system?

Thanks in advance.

**Comment by wopss** [comment, marked as answer]
Hello,

You also need to use Proton Tricks for RED4ext and add `winmm` to it. Maybe this [NexusMods](https://forum.nexusmods.com/index.php?showtopic=9916628/#entry121228491) post can help.

**Reply by jceloto7** [reply]
Hey WopsS

Iniatilly tks for the mod.

I followed this instructions and now I have no more problems. 

Thanks for the help :)

**Reply by wopss** [reply]
No problem! ❤️
